telegram/_utils/enum.py

```diff
@@ -23,30 +23,54 @@
     user. Changes to this module are not considered breaking changes and may not be documented in
     the changelog.
 """
-from enum import Enum
+import enum
+import sys
 from typing import Type, TypeVar, Union
 
 _A = TypeVar("_A")
 _B = TypeVar("_B")
-_Enum = TypeVar("_Enum", bound=Enum)
+_Enum = TypeVar("_Enum", bound=enum.Enum)
 
 
-def get_member(enum: Type[_Enum], value: _A, default: _B) -> Union[_Enum, _A, _B]:
+def get_member(_enum: Type[_Enum], value: _A, default: _B) -> Union[_Enum, _A, _B]:
```

Comment:
1. Nitpick: usually prefixing an argument with _ means that it's unused. In cases where there is an attempt to avoid name collisions it's customary to put the _ at the end.
2. The docstring needs updating.

And thinking as I write: maybe it would be more "clean" to do `import enum as _enum` at the top of the file.

Reply:
Doesn't prefixing with `_` means it's "private"? Afaik, unused is just `_`. So leave `import enum` as is?

Reply:
Funny how the same notation means different things.

When an argument to a function is prefixed with _ it means it is unused.

When a function (or an import) is named prefixed with _ it means it is private.